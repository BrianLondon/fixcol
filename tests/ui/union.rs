use fixed_derive::{ReadFixed};

#[derive(ReadFixed)]
#[repr(C)]
union MyUnion {
    f1: u32,
    f2: f32,
}

pub fn main() {}
