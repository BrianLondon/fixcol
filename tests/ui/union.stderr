error: fixed-derive error: Deriving ReadFixed on unions is not supported
 --> tests/ui/union.rs:5:1
  |
5 | union MyUnion {
  | ^^^^^

error: fixed-derive error: Deriving WriteFixed on unions is not supported
 --> tests/ui/union.rs:5:1
  |
5 | union MyUnion {
  | ^^^^^
